kotlin/app/src/main/java/com/fullstorydev/shoppedemo/data/Item.kt

```diff
@@ -0,0 +1,13 @@
+package com.fullstorydev.shoppedemo.data
+
+import androidx.room.ColumnInfo
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "items")
+data class Item (
```

Comment:
overall, I'm wondering if `CartItem` would be a better name. Overall, swapping `item` reference and definition names for `cartItem`. I'll approve this PR, but it might be a good idea to keep this in mind for a readability/interpretability enhancement.

Reply:
I think cartItem makes it more clear. Better refractor early than migrate later.